Anytype/Sources/PresentationLayer/Common/SwiftUI/BasicComponents/ButtonForeground.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import SwiftUI
+
+private struct ButtonForegroundViewModifier: ViewModifier {
+    
+    @Environment(\.isEnabled) private var isEnable
+    
+    func body(content: Content) -> some View {
+        content
+            .foregroundColor(isEnable ? .Button.active : .Button.inactive)
+    }
+}
+
+extension View {
+    func buttonDynamicForegroundColor() -> some View {
+        modifier(ButtonForegroundViewModifier())
+    }
+}
```

Comment:
Naming not really transparent on what to expect from `buttonDynamicForegroundColor`
Also `isEnabe` and `foregroundColor` can be applied not only to buttons but other components as well
I have two optional suggestions
rename `buttonDynamicForegroundColor` to `foregroundColorForDisabledState`
or make it 
dynamicForegroundColor(enabled: Color, disabled: Color)

Reply:
discussed